x-pack/plugins/observability/public/components/slo/slo_status_badge/slo_group_by_badge.tsx

```diff
@@ -5,24 +5,25 @@
  * 2.0.
  */
 
-import { EuiBadge, EuiFlexItem, EuiToolTip } from '@elastic/eui';
+import { EuiBadge, EuiBadgeProps, EuiFlexItem, EuiToolTip } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import { ALL_VALUE, SLOWithSummaryResponse } from '@kbn/slo-schema';
-import { euiLightVars } from '@kbn/ui-theme';
 import React from 'react';
+import { euiLightVars } from '@kbn/ui-theme';
 
 export interface Props {
+  color?: EuiBadgeProps['color'];
   slo: SLOWithSummaryResponse;
 }
 
-export function SloGroupByBadge({ slo }: Props) {
+export function SloGroupByBadge({ slo, color }: Props) {
```

Comment:
I was thinking passing the viewMode as a prop and checking against it to calculate the color.

Reply:
i think creating a new file seemed better to me. And color is only needed when used by cards, for list view, we use the color which is already defined.

Reply:
Btw we don't need to use the group_by badge for the card view. As we discussed with Maciej, we will use subheader for the group by.

Reply:
i have updated that.